tensorflow/core/kernels/sparse_fill_empty_rows_op.cc

```diff
@@ -94,6 +94,16 @@ void SparseFillEmptyRowsOpImpl(OpKernelContext* context,
   OP_REQUIRES_ASYNC(context, dense_shape_t.NumElements() != 0,
                     errors::InvalidArgument("Dense shape cannot be empty."),
                     done);
+  // Validation for negative dims inside `dense_shape`.
+  auto dense_shape_vec = dense_shape_t.vec<int>();
```

Comment:
I think this should be int64, since sparse tensors have 64-bit dense shapes.

Also can you add a test to [`sparse_ops_test.py`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/sparse_ops/sparse_ops_test.py)?